src/media/icons/mlem.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I just took a svg from Figma and tweaked it a bit. We'll eventually have to update our `static` images to include official MLEM images.